apps/examples/rdp/README.txt

```diff
@@ -0,0 +1,8 @@
+examples/rdp
+^^^^^^^^^^^^^^
+
+  This is the RDP Demo main example. This shows how to call
```

Comment:
What is the meaning of RDP? Could you explain in this README also?

Reply:
done